### PR TITLE
Fix eslint issues

### PR DESCRIPTION
**Pull request by chainergar**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -58,6 +54,17 @@
...
+    "react/no-array-index-key": "off",
+    "no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
+    "jsx-a11y/no-noninteractive-element-interactions": "off",
+    "jsx-a11y/no-static-element-interactions": "off",
+    "react/button-has-type": "off",
+    "no-shadow": "off",
+    "jsx-a11y/click-events-have-key-events": "off",
+    "no-restricted-syntax": "off",
+    "max-classes-per-file": "off",
+    "guard-for-in": "off"
```
Comment (cryptopower80) [inline .eslintrc:68]: can you please keep the same configuration? We will make sure we have a clean and good code

Comment (cryptopower80) [reply .eslintrc:68]: Resolved on discord

File: src/components/farmCards/FarmCardGroupContainer.jsx [hunk 2]
```diff
@@ -21,23 +22,21 @@ function FarmCardGroupContainer({ showAsTables }) {
...
 
   useEffect(() => {
-    if (!state.totalFarmEarned) {
-      getTotalFarmEarned();
-    }
-    return;
-    // eslint-disable-next-line 
+    getTotalFarmEarned();
+
+    // eslint-disable-next-line
```
Comment (cryptopower80) [inline src/components/farmCards/FarmCardGroupContainer.jsx:39]: would it be possible to know why we disable this rule?

File: src/components/harvest/Harvest.jsx [hunk 3]
```diff
@@ -61,17 +59,14 @@ const Harvest = () => {
         ? ethers.utils.parseUnits(state.minimumHarvestAmount.toString(), 18)
         : await pool.unstakedBalance(state.address);
 
-    console.log(
-      state.minimumHarvestAmount,
-      parseFloat(amount / 10 ** 18).toString(),
-    );
+    console.log(state.minimumHarvestAmount, parseFloat(amount / 10 ** 18).toString());
```
Comment (cryptopower80) [inline src/components/harvest/Harvest.jsx:62]: would it be possible to remove this debug log? I don't think it's necessary

File: src/lib/pool.js [hunk 4]
```diff
@@ -36,68 +35,21 @@ export class RewardsPool extends ethers.Contract {
...
+  // static fromPool(pool, provider) {
+  //   switch (pool.type) {
+  //     case 'autocompounding':
+  //       // eslint-disable-next-line no-use-before-define
+  //       return new AutoCompoundingRewardsPool(pool, provider);
+  //     default:
+  //       // eslint-disable-next-line no-use-before-define
+  //       return new HarvestRewardsPool(pool, provider);
+  //   }
+  // }
```
Comment (cryptopower80) [inline src/lib/pool.js]: Would it be possible to know through a comment why we commented out this piece?

File: src/lib/tokens/Token.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  // /**
+  //  *
+  //  * @param {Object} name
+  //  * @param {Object} provider web3 provider
+  //  * @return {Token} an Token subclass instances
+  //  */
+  // static fromName(name, provider) {
+  //   return getTokenFromAsset(data.assetByName(name), provider);
+  // }
+}
```
Comment (cryptopower80) [inline src/lib/tokens/Token.js:34]: same comment as above

File: src/lib/utils.js [hunk 6]
```diff
@@ -1,5 +1,7 @@
 import ethers from 'ethers';
 import { formatUnits } from 'ethers/lib/utils';
+// import AutoCompoundingRewardsPool from './AutoCompoundingRewardsPool';
+// import HarvestRewardsPool from './HarvestRewardsPool';
```
Comment (cryptopower80) [inline src/lib/utils.js]: I think we remove these these imports if we don't use them

File: src/components/farmCards/FarmCardGroupContainer.jsx (hunk 2 above)
Comment (chainergar) [reply src/components/farmCards/FarmCardGroupContainer.jsx:39]: resolved

File: src/components/harvest/Harvest.jsx (hunk 3 above)
Comment (chainergar) [reply src/components/harvest/Harvest.jsx:62]: resolved

File: src/lib/utils.js (hunk 6 above)
Comment (chainergar) [reply src/lib/utils.js]: this is outdated code
already resolved one

File: src/lib/tokens/Token.js (hunk 5 above)
Comment (chainergar) [reply src/lib/tokens/Token.js:34]: this is outdated code
already resolved one

File: src/lib/pool.js (hunk 4 above)
Comment (chainergar) [reply src/lib/pool.js]: this is outdated code
already resolved one

**Review by cryptopower80 — APPROVED** [review]

**Pull request merged**